examples/np/cem_cartpole.py

```diff
@@ -47,7 +47,7 @@ def cem_cartpole(ctxt=None, seed=1):
                    max_path_length=100,
                    n_samples=n_samples)
 
-        runner.setup(algo, env, sampler_cls=OnPolicyVectorizedSampler)
+        runner.setup(algo, env, sampler_cls=MultiprocessingSampler)
```

Comment:
what would the default be if you removed this? is it not okay?

Reply:
It is okay to remove this too. I think I'll replace it with `VecWorker` instead.